rest_framework/serializers.py

```diff
@@ -846,6 +843,9 @@ def get_related_field(self, model_field, related_model, to_many):
             if model_field.help_text is not None:
                 kwargs['help_text'] = model_field.help_text
 
+            if model_field.rel.limit_choices_to:
+                kwargs['limit_choices_to'] = model_field.rel.limit_choices_to
```

Comment:
I'm not clear about whether it should be `related_model` instead of `model_field.rel` here. Thoughts ?